### ISSUE TITLE
[Bug]: TextArea and others have incorrect focus outline color in high contrast mode

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#33135]
### Component

Textarea

### Package version

9.55.1

### React version

18.3.1

### Environment

 System:
    OS: Windows 11 10.0.22631
    CPU: (22) x64 Intel(R) Core(TM) Ultra 7 165H
    Memory: 33.99 GB / 63.64 GB
  Browsers:
    Edge: Chromium (127.0.2651.74)
    Internet Explorer: 11.0.22621.3527

### Current Behavior

Current focus outline is white

![Image](https://github.com/user-attachments/assets/ee8231aa-188b-4ca7-870f-5697e37f42dd)

### Expected Behavior

Outline should be the contrast one, for Aquatic theme the blue one.

### Reproduction

https://stackblitz.com/run?file=src%2Fexample.tsx

### Steps to reproduce

1. Switch the HC mode to Aguatic
2. Navigate to the affected component

**Others affected component:** Field, Combobox, Input, Dropdown, TagPicker, DatePicker, TimePicker

### Are you reporting an Accessibility issue?

None

### Suggested severity

Urgent - No workaround and Products/sites are affected

### Products/sites affected

Teams

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.
